doc/source/batch-inference/getting-started.rst

```diff
@@ -0,0 +1,44 @@
+Scalable Offline Batch Inference
+================================
+:ref:`Ray Data <datasets>` offers a highly performant and scalable solution for offline batch inference and processing on large amounts of data. 
```

Comment:
Quick paragraph on what batch inference is maybe? Also, we should somehow mention that there are many ways to run batch inference with Ray, but we're starting with the most convenient here (or so).